Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -58,6 +58,113 @@ public void Settings_ShouldStoreSettingsAndFeatureFlags(string featureName)
         }
     }
 
+    [Test]
+    [Category("Actions")]
+    public void Actions_WithMultipleBindingsAndMultipleInteractions_Works()
+    {
+        InputSystem.settings.defaultButtonPressPoint = 0.5f;
+
+        var keyboard = InputSystem.AddDevice<Keyboard>();
+
+        var action = new InputAction(interactions: "tap,hold(duration=2)");
+        action.AddBinding("<Keyboard>/w");
+        action.AddBinding("<Keyboard>/a");
+        action.Enable();
+
+        IInputInteraction performedInteraction = null;
+        IInputInteraction canceledInteraction = null;
+        action.performed += ctx =>
+        {
+            performedInteraction = ctx.interaction;
+        };
+        action.canceled += ctx =>
+        {
+            canceledInteraction = ctx.interaction;
+        };
+
+        // PressRelease AW trigger a tap
+        currentTime = 0;
+        InputSystem.QueueStateEvent(keyboard, new KeyboardState(Key.A, Key.W));
+        InputSystem.Update();
+
+        // nothing triggered
+        Assert.That(canceledInteraction, Is.Null);
+        Assert.That(performedInteraction, Is.Null);
+
+        currentTime = 0.01;
```

Comment:
Minor, maybe have 

// Release A, W

as comment to follow the pattern here for readability?